### DISCUSSION TITLE
Combobox that accepts user typed value

### DISCUSSION THREAD

**Discussion by HarrisWebApps** [tailwindlabs/headlessui#1127] [Ideas]
I very much appreciate the new Combobox with autocomplete.   Great work!!

Just curious if there is an easy way to make it work with any value. (i.e. so the user can type any value and that it becomes the selected value).  I see Vuetify has a combobox component that allows any such value:  [Vuetify - Combobox](https://vuetifyjs.com/en/components/combobox/)

Semantically, I think a combo box is supposed to be editable and accept a custom value as distinguished from a select or drop-down.  See:  [Combo box defined](https://en.wikipedia.org/wiki/Combo_box)

**Comment by nmoncur** [comment]
Throwing my vote onto the pile as well. A combo box is just a searchable select if it can't accept a custom value.

**Reply by nmoncur** [reply]
My googling continues and I might have spoken too soon, ha. https://headlessui.com/vue/combobox#allowing-custom-values
